motr/io_req.c

```diff
@@ -1474,6 +1474,7 @@ static int ioreq_dgmode_read(struct m0_op_io *ioo, bool rmw)
 
 	M0_LOG(M0_DEBUG, "[%p] dgmap_nr=%u is in dgmode",
 			 ioo, ioo->ioo_dgmap_nr);
+	ioo->ioo_pbuf_type = M0_PBUF_DIR;
```

Comment:
Add comment here about the need for setting this flag.

Reply:
Added comment.

Reply:
@yatin-mahajan Could you also update the comments in enum m0_pbuf_type for M0_PBUF_DIR adding a third case?